worker-languagedetection-container/testcases/automated/CAF_4001/ganda.yaml.testcase

```diff
@@ -30,39 +30,39 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "lg"
+        - data: "unknown"
```

Comment:
SIgnificant change here, cannot identify ganda anymore. If it is not supported then remove this testcase and make sure that ganda is not referred to as a supported language.